jipdate.py

```diff
@@ -227,7 +232,7 @@ def parse_status_file(jira, filename):
     # [SWG-28]
     # [LITE-32]
     # etc ...
-    regex = r"^\[([A-Z]+-\d+)\]\n$"
+    regex = r"^\[(.*)\]\n$"
```

Comment:
I hope so.  @jbech-linaro we should find this out.

So, more generally, if I understand correctly, this change is to let you have other sections following the progress update, because if you put the progress update in the bottom of the e-mail, then you could have all the [MISC] above that you want.

I was fighting with a similar challenge, because my weekly e-mails look like

Progress:
 - Worked more on [VIRT-123] and did some more hacking
 - Reviewed too much code

Plans:
 - Work more on [VIRT-789]

Others:
 - Holiday next week (not really)

And I wasn't sure how to combine that, with this, but maybe your suggestion works, as long as we can resolve something to be a jira issue - or not.